ssl-config/src/test/resources/readme.md

```diff
@@ -0,0 +1,65 @@
+# Test Keys and Certificates
+
+This directory contains certificates and keys in various formats that are used to test the package.
```

Comment:
do you have the scripts handy that you used to generate the trust/key stores from the KEY/PEM files?